package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "hamilton-svelte-template",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "author": "Hamilton Ulmer",
+  "license": "MPL-2.0",
```

Comment:
This is tricky, but I like what the [sveltejs/template package.json](https://github.com/sveltejs/template/blob/d78108ca29e61e31347e8dbf0a2c2f25bdafd4f5/package.json) does. It omits most of these and uses a generic name and version. Perhaps something like this?

```json
  "name": "graph-paper-app",
  "version": "1.0.0",
```

Reply:
Yep, good catch. I just transferred something I had in my personal repositories over here w/o changing anything. Feel free to change this to something generic.

Reply:
https://github.com/graph-paper-org/app-template/pull/7